WD-Firmware/psp/api_inc/psp_basic_macros.inc

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
change file name to psp_core_base.inc

Reply:
done

Reply:
done

Reply:
https://github.com/westerndigitalcorporation/riscv-fw-infrastructure/pull/12/commits/6c26f8aba68f07621c53b564702cbab8a7d91dd1